### PR TITLE
fix: Adding back functionality for rejectUnauthorized http option

### PR DESCRIPTION
**Pull request by jdalrymple**

### CODE REVIEW

**Comment by jdalrymple** [comment]
Latest change should handle this: https://github.com/node-gitlab/node-gitlab/issues/357#issuecomment-567528152

@casz give it a quick look :pray:

**Comment by casz** [comment]
conflicts in package.json 😱

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "gitlab",
   "description": "Full NodeJS implementation of the GitLab API. Supports Promises, Async/Await.",
-  "version": "14.0.0",
+  "version": "14.0.1",
```
Comment (casz) [inline package.json]: seems odd that this is touching version 🤔

Comment (jdalrymple) [reply package.json]: Just because i was pushing to a beta tag on npm

**Pull request merged**